### DISCUSSION TITLE
mailcheck.js to avoid email field typos

### DISCUSSION THREAD

**Discussion by jameswilson** [react-hook-form/react-hook-form#4236] [Q&A]
Is it possible to use [`mailcheck.run()`](https://github.com/mailcheck/mailcheck) with react-hook-form not for *validating* email fields, but for suggesting fixes.

**Comment by bluebill1049** [comment]
not sure, I haven't used it

**Reply by jameswilson** [reply]
Thanks for the quick response. 

Following up...  I found https://www.npmjs.com/package/react-mailcheck

And the example code on that page suggests `<MailCheck/>` is just a wrapper component with the `<input type="email">` inside of it, and so maybe it doesn't interfere with the `ref` attribute, so I think this could be done.

Will give it a try and report back.
